### PR TITLE
benchmark: client side workloads

### PR DESCRIPTION
**Pull request by zpencer**
Add missing client side workloads

### CODE REVIEW

File: benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java [hunk 1]
```diff
@@ -538,4 +577,89 @@ public void onClose(Status status, Metadata trailers) {
       }
     }
   }
+
+  /**
+   * Used for both STREAMING_FROM_CLIENT as well as STREAMING_BOTH_WAYS.
+   */
+  private class StreamingWorker<ReqT, RespT>  implements Runnable {
```
Comment (carl-mastrangelo) [inline benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java]: final

also remove extra space.

File: benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java [hunk 2]
```diff
@@ -538,4 +577,89 @@ public void onClose(Status status, Metadata trailers) {
...
+        }
+
+        @Override
+        public void onNext(RespT value) {
+          // noop
+        }
+
+        @Override
+        public void onError(Throwable t) {
+          call.cancel("onError called on client", null);
```
Comment (carl-mastrangelo) [inline benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java]: print the error?

File: benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java [hunk 3]
```diff
@@ -538,4 +577,89 @@ public void onClose(Status status, Metadata trailers) {
...
+
+        @Override
+        public void onCompleted() {
+          call.halfClose();
+        }
+      });
+    }
+  }
+
+  private class ServerStreamingWorker<ReqT, RespT> implements Runnable {
```
Comment (carl-mastrangelo) [inline benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java]: final

always final.

File: benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java [hunk 4]
```diff
@@ -538,4 +577,89 @@ public void onClose(Status status, Metadata trailers) {
...
+        @Override
+        public void onNext(RespT value) {
+          if (shutdown) {
+            call.cancel("shutting down", null);
+          }
+        }
+
+        @Override
+        public void onError(Throwable t) {
+          call.cancel("onError called on client", null);
```
Comment (carl-mastrangelo) [inline benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java]: include error here too.

File: benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java (hunk 2 above)
Comment (zpencer) [inline benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java]: done

File: benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java (hunk 3 above)
Comment (zpencer) [inline benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java]: done

File: benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java (hunk 4 above)
Comment (zpencer) [inline benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java]: done

**Pull request closed without merging**